src/script/conversation/linkPreviews/helpers.test.ts

```diff
@@ -73,10 +73,16 @@ describe('getFirstLinkWithOffset', () => {
     const singleTick = 'cool code: `wire.com`';
     const moreTicks = 'cool code: ```\nwire.com\n```';
     const manyTicks = 'cool code: ``````\nwire.com\n``````';
+    const multilineSingleTick = '\ncool code: `wire.com`';
+    const noClosingTicks = 'cool code: ```\nwire.com';
+    const ticksception = "cool code: ```\nwire.com `it's a trap!`\n```";
```

Comment:
can you try a payload that has both a real link AFTER a link inside markdown (and see if the function detects the real link)?

Reply:
oh yes, forgot! I did manually test it though, I'll add the test

Reply:
Here are some payloads that we shoud try 

multiline support
```
  ```
  code: `https://wire.com`
  some code
  ```
```

some real links before the markdown
```
  https://wire.com
  ```
  code: `https://wire.com`
  some code
  ```
```

some real links after the markdown
```
  ```
  code: `https://wire.com`
  some code
  ```
  https://wire.com
```